### ISSUE TITLE
The result of `getFieldsMap` should be serialized

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#3429]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
SearchResponse {Response {code = SUCCESS, message = "Success!"}, returned results for 5 queries}
 size of getFieldsMap = 50
```

**Steps/Code to reproduce behavior**
```
    @Test(dataProvider = "Collection", dataProviderClass = MainClass.class)
    public void testSearchCollection(MilvusClient client, String collectionName)  {
        InsertParam insertParam = new InsertParam.Builder(collectionName).withFields(Constants.defaultEntities).build();
        InsertResponse res = client.insert(insertParam);
        assert(res.getResponse().ok());
        List<Long> ids = res.getEntityIds();
        client.flush(collectionName);
        SearchParam searchParam = new SearchParam.Builder(collectionName).withDSL(dsl).build();
        SearchResponse res_search = client.search(searchParam);
        Assert.assertEquals(res_search.getResultIdsList().size(), Constants.nq);
        Assert.assertEquals(res_search.getResultDistancesList().size(), Constants.nq);
        Assert.assertEquals(res_search.getResultIdsList().get(0).size(), Constants.topk);
        Assert.assertEquals(res_search.getResultIdsList().get(0).get(0), ids.get(0));
        Assert.assertEquals(res_search.getFieldsMap().get(0).size(), top_k);
    }
```

**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by sahuang** [comment]
Fixed in 0.9.0-SNAPSHOT @del-zhenwu

**Comment by del-zhenwu** [comment]
close
